oaknational/Oak-Web-Application
# fix: address console warnings

## Description
- Refactored code to remove warning about nested styled component in subject phase picker
- Refactored code to remove warning about list key in subject principles on overview tab

## How to test

1. Checkout branch `fix/curric-console-warning`
2. Run server with `npm run dev`
3. You should NOT see the warnings about nested style components and missing key in lists

## Checklist
